src/benchmark_environments/util.py

```diff
@@ -17,3 +22,183 @@ def step(self, action):
             info["terminal_observation"] = obs
             obs = self.env.reset()
         return obs, rew, False, info
+
+
+class AbsorbAfterDoneWrapper(gym.Wrapper):
+    """Goes to absorb state after done=True. Always returns done=False.
```

Comment:
```suggestion
    """Transition into absorbing state instead of episode termination.
    
    When the environment being wrapped returns `done=True`, we return an absorbing observation.
    This wrapper always returns `done=False`.
```